dogfood/coder-envbuilder/main.tf

```diff
@@ -5,7 +5,7 @@ terraform {
     }
     docker = {
       source  = "kreuzwerker/docker"
-      version = "~> 3.0.0"
+      version = "~> 3.0"
```

Comment:
Allows minor version bumps.

Reply:
@matifali I think this will allow bumping from `3.0` all the way to `3.9`, is this what we want?
I thought our modules were in `a.b.c` format?

Reply:
>  I think this will allow bumping from 3.0 all the way to 3.9, is this what we want?

Yes. Minor updates are non-breaking, so they are safer and can bring new bug fixes and overall improvements.

Reply:
Docker provider is already on 3.6, and we have many bugs fixed.

Reply:
my bad, I thought we considered `b` to be a major version